Source/MediaStorageAndFileFormat/gdcmUIDGenerator.cxx

```diff
@@ -112,17 +111,38 @@ struct fnv_hash
     }
 };
 
-const char* UIDGenerator::Generate()
+/*
+Generates a UID according to ISO/IEC 9834-8 / ITU-T X.667 / RFC4122
+http://dicom.nema.org/Dicom/2013/output/chtml/part05/sect_B.2.html
```

Comment:
The link "http://dicom.nema.org/Dicom/2013/output/chtml/part05/sect_B.2.html" appears to be broken.

Reply:
Strange, it is working for me

Reply:
lol, now it is working for me as well...